bundle.yml

```diff
@@ -69,14 +69,13 @@ projects :
             ATLAS_ENABLE_CUDA=OFF
 
     - field_api :
-        git     : ${BITBUCKET}/rdx/field_api
-        version : master
+        git     : https://github.com/ecmwf-ifs/field_api.git
+        version : main
         optional: true
         require : ecbuild
         cmake   : >
-            ENABLE_FIELD_API_TESTS=OFF
-            ENABLE_FIELD_API_FIAT_BUILD=OFF
-            FIELD_API_UTIL_MODULE_PATH=${CMAKE_SOURCE_DIR}/cloudsc-dwarf/src/common/module
+            ENABLE_SINGLE_PRECISION=OFF
```

Comment:
Since this is effectively a global option in this form, we might want to simply put this at the top of the bundle.yml.